apps/wallet/src/background/account-sources/index.ts

```diff
@@ -29,11 +29,11 @@ function toAccountSource(accountSource: AccountSourceSerialized) {
 	throw new Error(`Unknown account source of type ${accountSource.type}`);
 }
 
-export async function getAccountSources(filter?: { type: AccountSourceType }) {
+export async function getAccountSources(filter?: { type: AccountSourceType[] }) {
```

Comment:
it seems that we don't use this change, change type to array so not sure if we should keep it